src/test/java/blackjack/domain/HandTest.java

```diff
@@ -0,0 +1,38 @@
+package blackjack.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class HandTest {
+
+    @DisplayName("핸드를 생성한다")
+    @Test
+    public void createSuccess() {
+        assertThatCode(Hand::new)
+                .doesNotThrowAnyException();
+    }
+
+    @DisplayName("패를 추가한다")
+    @Test
+    public void addCard() {
+        Hand hand = new Hand();
+
+        hand.put(CardFixture.heartJack());
+
+        assertThat(hand.getCards().size()).isEqualTo(1);
```

Comment:
```suggestion
        assertThat(hand.getCards()).hasSize(1);
```
컬랙션의 크기를 검증할 때를 위한 assertJ api 를 적극 활용해보죠!

Reply:
저런 메서드도 있었네요. JUInit 및 AssertJ 를 더 학습해야겠어요.